src/blurredLocation.js

```diff
@@ -62,6 +61,8 @@ BlurredLocation = function BlurredLocation(options) {
     });
     var geometry = Blurred.responseJSON.results[0].geometry.location;
     options.map.setView([geometry.lat, geometry.lng],options.zoom);
+    $("#lat").val(geometry.lat);
```

Comment:
Actually there is a different method which pans the map to geocode location namely panMapToGeocodedLocation also geocode only return the location and fills up lat and lon input listeners which can be change to some other method input by options